es-app/src/guis/GuiGamelistOptions.cpp

```diff
@@ -64,7 +65,9 @@ GuiGamelistOptions::GuiGamelistOptions(Window* window, SystemData* system) : Gui
 	mMenu.addRow(row);
 
 	std::map<std::string, CollectionSystemData> customCollections = CollectionSystemManager::get()->getCustomCollectionSystems();
-	if((customCollections.find(system->getName()) != customCollections.end() && CollectionSystemManager::get()->getEditingCollection() != system->getName()) ||
+
+	if(ViewController::get()->isUIModeFull() &&
```

Comment:
So, here's an unintended behavior.

While I'd be happy to consider "add/remove" from a collection the same functionality as "favorites", I understand if you want to keep just "favorites" for the Kiosk mode. I won't object one way or the other.

Still, for what you currently intend to do, you need to add an extra set of brackets to encompass 

```
(customCollections.find(system->getName()) != customCollections.end() && CollectionSystemManager::get()->getEditingCollection() != system->getName()) || CollectionSystemManager::get()->getCustomCollectionsBundle()->getName() == system->getName()
``` 

otherwise it will work for the "My Collections" bundle but not for any other collection at the top level :) Just ran into this and thought I'd give you the heads up.

I'm using this as my main ES for the time being, so I'll let you know if anything else comes up :)

Reply:
Thanks for this! My intention was indeed to keep add/remove from collection **out** of the kiosk mode. I will add the brackets to fix it for all collection types.